src/java.cpp

```diff
@@ -14,7 +14,8 @@
 
 #define DYNAMIC_PROXY_JS_ERROR -4
 
-long v8ThreadId;
+long v8ThreadIdWIN32;
```

Comment:
I was hoping that by using the typedef you could get rid of a lot of the edits below. For example `my_getThreadId` could just be written:

```
v8ThreadId my_getThreadId() {		
  #ifdef WIN32
    return GetCurrentThreadId();
  #else
    return pthread_self();
  #endif
}
```

then we should be able to get rid of the isWIN32 logic all together.

If you want to use `pthread_equal` could `#define v8ThreadIdEquals(a,b) ...`